<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWin</class>
 <widget class="QWidget" name="LoginWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>354</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stack_wid_main">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="main_page">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wid_top" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>130</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>130</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#wid_top
{
background-color: qlineargradient(spread:reflect, x1:0.845, y1:0, x2:0.408, y2:0.829727, stop:0 rgba(0, 20, 0, 255), stop:0.966102 rgba(102, 112, 255, 255));
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,10">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="wid_tool" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,10,4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lab_logo">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/icon/Win_Login/logo.png);
margin-top:8px;
margin-bottom:8px;
margin-left:8px;
margin-right:8px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lab_logo_text">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/icon/Win_Login/logo_text.png);
margin-top:10px;
margin-bottom:10px;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>197</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="wid_top_btns" native="true">
               <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="10,10,10">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QPushButton" name="btn_setting">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">

QPushButton 
{
border-image: url(:/icon/Win_Login/setting.png);
}

QPushButton:pressed
{
border-image: url(:/icon/Win_Login/setting.png);
background-color: rgb(255, 255, 255,200);
}
QPushButton:hover
{
border-image: url(:/icon/Win_Login/setting.png);
background-color: rgb(255, 255, 255,100);
}


QPushButton:disabled 
{

}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="btn_min">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">

QPushButton 
{
border-image: url(:/icon/Win_Login/win_min.png);
}

QPushButton:pressed
{
border-image: url(:/icon/Win_Login/win_min.png);
background-color: rgb(255, 255, 255,200);
}
QPushButton:hover
{
border-image: url(:/icon/Win_Login/win_min.png);
background-color: rgb(255, 255, 255,100);
}


QPushButton:disabled 
{

}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="btn_close">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
QPushButton 
{
border-image: url(:/icon/Win_Login/close.png);
}

QPushButton:pressed
{
border-image: url(:/icon/Win_Login/close.png);
background-color: rgb(255, 84, 57,255);
}
QPushButton:hover
{
border-image: url(:/icon/Win_Login/close.png);
background-color: rgb(255, 84, 57,255);
}


QPushButton:disabled 
{

}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>22</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>77</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wid_bottom" native="true">
         <property name="styleSheet">
          <string notr="true">#wid_bottom
{
background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,10,10">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="wid_left" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>175</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_register">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(166, 166, 166);
border:0px;
</string>
               </property>
               <property name="text">
                <string>注册账号</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wid_mid" native="true">
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>240</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>40</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QWidget" name="wid_account" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#wid_account
{
border:1px solid;
border-top:0px;
border-left:0px;
border-right:0px;
border-color:rgb(229,229,229);
}</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
                <property name="spacing">
                 <number>5</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="lab_account_logo">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/icon/Win_Login/uncheck_account_ logo.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="edit_account">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:0px;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="wid_password" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1111111</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#wid_password
{
border:1px solid;
border-top:0px;
border-left:0px;
border-right:0px;
border-color:rgb(229,229,229);
}</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,100">
                <property name="spacing">
                 <number>5</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="lab_password_logo">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/icon/Win_Login/uncheck_code_logo.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="edit_password">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>11111</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:0px;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="wid_btns" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="checkbtn_auto_login">
                  <property name="styleSheet">
                   <string notr="true">QCheckBox{color:rgb(166,166,166)} 

QCheckBox::indicator {width: 15px; height:15px;color: rgb(166, 166, 166);}

QCheckBox:focus
{
color:rgb(166, 166, 166);
}

QCheckBox::indicator:unchecked {border-image: url(:/icon/Win_Login/uncheck_checkbtn.png);}


QCheckBox::indicator:checked {border-image: url(:/icon/Win_Login/check_checkbtn.png);}
</string>
                  </property>
                  <property name="text">
                   <string>自动登陆</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbtn_remb_password">
                  <property name="styleSheet">
                   <string notr="true">QCheckBox{color:rgb(166,166,166)} 

QCheckBox::indicator {width: 15px; height:15px;color: rgb(166, 166, 166);}

QCheckBox:focus
{
color:rgb(166, 166, 166);
}

QCheckBox::indicator:unchecked {border-image: url(:/icon/Win_Login/uncheck_checkbtn.png);}


QCheckBox::indicator:checked {border-image: url(:/icon/Win_Login/check_checkbtn.png);}
</string>
                  </property>
                  <property name="text">
                   <string>记住密码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_search_password">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(166, 166, 166);
border:0px;</string>
                  </property>
                  <property name="text">
                   <string>找回密码</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(80, 90, 196);
radius:5px;
border-radius:5px;</string>
               </property>
               <property name="text">
                <string>登陆</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wid_right" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="panda_chat.qrc"/>
 </resources>
 <connections/>
</ui>
